# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/glib-networking/no-tests.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glib-networking-2.68.1/meson.build.vanilla	2021-07-22 12:28:12.650227764 +0200
+++ glib-networking-2.68.1/meson.build	2021-07-22 12:30:47.980235777 +0200
@@ -189,7 +189,7 @@
   subdir('tls/openssl')
 endif
 
-subdir('tls/tests')
+#subdir('tls/tests')
 
 # Will automatically pick it up from the cross file if defined
 gio_querymodules = find_program('gio-querymodules', required : false)
